From 54ece69109afc75e2e229f75280f0f7ae1b0aa61 Mon Sep 17 00:00:00 2001
From: Varun Sharma <vsharma@habana.ai>
Date: Mon, 20 Nov 2023 12:20:28 +0200
Subject: [PATCH] [SW-156330] gaudi2: remove erom tpm bin from install script

This patch remove erom tpm bin as requirement when package
BR2_PACKAGE_PREBOOT_ARC_EXT_ROM is enabled, as now tpm bin
is not part of erom.

Change-Id: Iddc478855a2edba0c693a5bd3566d6a081f029a9
Signed-off-by: Varun Sharma <vsharma@habana.ai>
---

diff --git a/package/preboot/preboot.mk b/package/preboot/preboot.mk
index 4b79191..38aaa9c 100644
--- a/package/preboot/preboot.mk
+++ b/package/preboot/preboot.mk
@@ -42,7 +42,6 @@
 	if [ "$(BR2_PACKAGE_PREBOOT_ARC_EXT_ROM)" == "y" ]; then \
 		$(INSTALL) -D -m 0644 $(@D)/app/ext_rom/erom_recovery.bin $(BINARIES_DIR)/erom_recovery.bin; \
 		$(INSTALL) -D -m 0644 $(@D)/app/ext_rom/erom_pcie.bin $(BINARIES_DIR)/erom_pcie.bin; \
-		$(INSTALL) -D -m 0644 $(@D)/app/ext_rom/erom_tpm.bin $(BINARIES_DIR)/erom_tpm.bin; \
 	fi;
 
 	if [ "$(BR2_PACKAGE_PCI_PIF_ARC)" == "y" ]; then \
